docs/usage/automerge-configuration.md

```diff
@@ -0,0 +1,111 @@
+I think this is a good usecase to cover, it's something that a lot of people will probably want.
+
+### Automerge Prettier or ESLint dependency
+
+This is probably a good one to cover as well.
+I think many users just click "merge" on those types of updates as well.
+
+### Automerge patch/minor update
+
+Automerge patch level updates is probably good to cover as well.
+Patch level `minor` updates shouldn't have breaking changes (if they follow SemVer that is...), so that makes them a good case to cover as well.
+
+### Automerging and scheduling
+
+Automerging is particularly beneficial if you have configured a schedule, because Renovate on its own may be able to automerge the majority of your updates.
+And this is especially so if your repository needs rebasing, e.g. because you use lock files. e.g. let's say you have dependencies `abc` and `xyz` with upgrades, and you use a `yarn.lock` file.
+
+- At the start of the schedule, `Renovate` will create branches for `abc` and `xyz` upgrades, including `yarn.lock` updates
+- After `abc` passes tests, `Renovate` will automerge it to `master`
+- The `xyz` branch probably now has `yarn.lock` conflicts
+- Renovate will immediately check all other branches and rebase them
+- The change to `xyz` branch will trigger another round of CI tests
+- After the updated `xyz` branch passes, Renovate will automerge it too
+
+This is a lot better than you waking up to two PRs and then having to deal with conflicts yourself after you merge the first one.
+
+Remember our running `eslint` example? Let's automerge it if all the linting updates pass:
+
+```json
+  "packageRules": [
+    {
+      "packagePatterns": [ "eslint" ],
+      "groupName": "eslint",
+      "schedule": ["before 2am on monday"],
+      "automerge": true,
+      "automergeType": "branch"
+    }
+  ]
+```
+
+Have you come up with a rule that you think others would benefit from?
+How about a PR back to [renovate-config](https://github.com/singapore/renovate-config) with the above rule named `":automergeEslintWeekly"` ?
+
+## Frequent problems and how to resolve them
+
+### Automerge config not correct to begin with
+
+What are the most common mistakes people make when doing the initial configuring?
+
+### Branch vs PR
+
+Those of you familiar with GitHub might note that even if you automerge PRs, you are still going to get notifications (noise) anyway - one when the PR is created and another when it is merged.
+For this reason we recommend you consider setting `automergeType=branch` which will mean:
+
+- Renovate first creates a branch and no PR
+- If tests pass, Renovate pushes a commit directly to `master` without PR
+- If tests fail, Renovate raises a PR for you to review
+
+The result is that passing updates are essentially "silent" - the only sign of them are the commits to your `master` branch.
+
+### Absence of tests
+
+If you have no tests but still want Renovate to automerge, you need to add "requiredStatusChecks": null to your configuration.
```

Comment:
We should note that this is a really bad idea to automerge without Tests.

Reply:
I agree that it's silly to automerge if you don't have good test coverage.
How about something like this instead?

```suggestion
We recommend that you only turn on automerge once you have good test coverage.
Having good test coverage will prevent you from automerging dependencies that break your build.

If you still want to turn on automerge, despite having no or lackluster tests, you can add "requiredStatusChecks": null to your configuration.
```

Reply:
Rhys made some changes to the text, so I'm assuming that's the way it should be worded.